integration-tests/validation/src/main/java/com/android/designcompose/testapp/validation/examples/StateCustomizations.kt

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.android.designcompose.testapp.validation.examples
+
+import android.annotation.TargetApi
+import android.os.Handler
+import android.os.Looper
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableFloatStateOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import com.android.designcompose.MeterState
+import com.android.designcompose.annotation.Design
+import com.android.designcompose.annotation.DesignComponent
+import com.android.designcompose.annotation.DesignDoc
+import java.time.Clock
+import java.time.LocalTime
+import java.time.format.DateTimeFormatter
+
+@DesignDoc(id = "POWyniB6moGRmhZTJyejwa")
+interface StateCustomizations {
+    @DesignComponent(node = "#root")
+    fun root(
+        @Design(node = "#time") time: androidx.compose.runtime.State<String>,
+        @Design(node = "#firstHand") firstHand: MeterState,
+        @Design(node = "#secondHand") secondHand: MeterState
+    )
+}
+
+@Composable
+@TargetApi(26)
+fun StateCustomizationsTest(clock: Clock) {
+    val time = remember { mutableStateOf("00:00:00") }
+    val firstHand = remember { mutableFloatStateOf(0F) }
+    val secondHand = remember { mutableFloatStateOf(0F) }
+
+    val handler = Handler(Looper.getMainLooper())
+    val runnable =
+        object : Runnable {
+            override fun run() {
+                val currentTime = LocalTime.now(clock)
```

Comment:
This means the example will show different results depending on the time, correct? I think it'd be better to have it use a specified time so that you get the same results whenever you run it.

Reply:
It depends on the Clock passed to the StateCustomizationsTest. The one in validation app will show the device time while the roborazzi test uses a fixed time clock that we get the same results whenever we run it.

Reply:
Ok, I see it now. Looks good.